### PR TITLE
Host file system read only

### PR DESCRIPTION
**Pull request by mylife6585**
the query add partition change detect
severity level change to critical

### CODE REVIEW

File: _data/rules.yml [hunk 1]
```diff
@@ -223,6 +223,10 @@ groups:
                 description: '{{ $labels.instance }} interface {{ $labels.device }} has encountered {{ printf "%.0f" $value }} transmit errors in the last five minutes.'
                 query: 'increase(node_network_transmit_errs_total[5m]) > 0'
                 severity: warning
+              - name: Host filesystem readonly
+                description: '{{ $labels.instance }} partition {{ $labels.device }} has become readonly.'
+                query: 'node_filesystem_readonly > 0 AND changes(node_filesystem_readonly[5m]) > 0'
```
[the remark this answers is not in the document]
Comment (samber) [reply _data/rules.yml:228]: Why did you add the `changes(node_filesystem_readonly[5m]) > 0` condition ?

The alert would fire then pass to green. On-call sysadmin might assume that problem is solved. 🤔

[the remark this answers is not in the document]
Comment (mylife6585) [reply _data/rules.yml:228]: > Why did you remove filesystem exclusion which was present in #114? That was a good list.

After adding change condition, the file system similar to tmpfs, even if it's readonly, will not trigger an alarm, so I removed it

[the remark this answers is not in the document]
Comment (mylife6585) [reply _data/rules.yml:228]: > Why did you add the `changes(node_filesystem_readonly[5m]) > 0` condition ?
> 
> The alert would fire then pass to green. On-call sysadmin might assume that problem is solved. 🤔

Yes, I have thought about this problem, but if it is not added, some file systems that should be readonly will fire.

[the remark this answers is not in the document]
Comment (mylife6585) [reply _data/rules.yml:228]: maybe changes(node_filesystem_readonly[1h]) > 0 will be better?
On-call sysadmin will have enough time to find and deal with the problem

[the remark this answers is not in the document]
Comment (samber) [reply _data/rules.yml:228]: IMO the good way to monitor this, is to exclude those FS from labels.

[the remark this answers is not in the document]
Comment (paulfantom) [reply _data/rules.yml:228]: > Why did you add the changes(node_filesystem_readonly[5m]) > 0 condition?

You're right. This should be something other than `changes()`.

[the remark this answers is not in the document]
Comment (mylife6585) [reply _data/rules.yml:228]: I change the description to 'is' ,and add back the exclude FS

**Pull request closed without merging**